addons/website_sale_stock/data/template_email.xml

```diff
@@ -11,6 +11,12 @@
             </div>
             <div style="display: flex; flex-direction: row; align-items: center; justify-content: center; width: 100%;">
                 <p t-esc="wishlist.product_id.name"/>
+                <p t-if="wishlist.product_id.product_template_attribute_value_ids"
+                   style="margin-left: 0.5em;">
+                    (<p
+                    t-out='wishlist.product_id.product_template_attribute_value_ids.name'
+                    />)
+                </p>
```

Comment:
Why this change by the way?

Reply:
https://github.com/odoo/odoo/pull/87789#discussion_r884882621